composer.json

```diff
@@ -13,26 +13,26 @@
   "minimum-stability": "stable",
   "prefer-stable": true,
   "require": {
-    "php": ">=7.4",
+    "php": ">=8.1",
```

Comment:
We cannot remove 7.4 support until we make a new major release of Terminus. Keep 7.4 for now.

Reply:
For the 4.x branch as per standup.